eclass/coreos-kernel.eclass

```diff
@@ -132,8 +132,10 @@ shred_keys() {
 }
 
 coreos-kernel_src_unpack() {
+	# we more or less reproduce the layout in /lib/modules/$(uname -r)/
 	mkdir -p "${S}/build" || die
-	ln -s "${KERNEL_DIR}"/* "${S}/" || die
+	mkdir -p "${S}/source" || die
```

Comment:
I kept it this way to ensure any potential ../ traversals would end up in ${S}, if we symlink the source tree a ../ traversal would end up in the parent of the link target.